core/src/types.ts

```diff
@@ -116,7 +116,7 @@ declare interface MarketDataBase extends Partial<ExchangeFees> {
 export declare interface Pool {
     addresses: string[];
     fee: number;
-    routes: string[];
+    routes: (string | undefined)[];
```

Comment:
In which case it can be an array of `undefined`? 😓

Reply:
uniswap autorouter does not promise to return the symbol of the token. It's optional there.